boards/BMS/Inc/Io/Io_StackWaterMark.h

```diff
@@ -1,16 +1,17 @@
 #pragma once
 
-#include "App_CanTx.h"
+struct BmsCanTxInterface;
 
 /**
- * Init the StackWaterMark monitor
- * @param can_tx_interface
+ * Register the CAN TX interface to be used in stack high watermark error
```

Comment:
Similar to https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/607/files#r424106251, with some additional function name and comment cleanup.